style.css

```diff
@@ -458,3 +483,534 @@
   }
 }
+
+@media only screen and (min-width: 678px) {
+  html {
+    background: #e5e5e5;
+  }
+
+  .body {
+    margin: 0;
+  }
+
+  .title {
+    font-family: Poppins-Bold, sans-serif;
+    font-weight: 700;
+    font-size: 48px;
+    color: #091e42;
+    padding-left: 10vw;
+  }
+
+  .header {
+    display: flex;
+    background: white;
+    width: 100vw;
+    justify-content: space-between;
+  }
+
+  .header nav img {
+    display: none;
+  }
+
+  .header-menu {
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    margin-top: 0;
+    margin-bottom: 0;
+    padding-left: 55vw;
+    padding-top: 20px;
+  }
+
+  .header-menu li {
+    font-family: Poppins, sans-serif;
+    color: #344563;
+    font-weight: 500;
+    font-size: 15px;
+    padding-left: 40px;
+  }
+
+  .brand {
+    font-family: Poppins-Bold, sans-serif;
+    max-height: 5%;
+    width: 90px;
+    padding-top: 2vh;
+    margin-bottom: 0;
+    padding-left: 5%;
+    text-decoration: none;
+    color: #6070ff;
+  }
+
+  .headline-container {
+    display: flex;
+    flex-direction: column;
+    height: 916px;
+    width: 100vw;
+    background-image: url("./resources/Desktop-version/images/Header\ bg@2x.png");
+    background-position: center;
+    background-size: 95%;
+    background-repeat: no-repeat;
+    background-color: white;
+    border-bottom-left-radius: 15%;
+  }
+
+  .article {
+    display: flex;
+    flex-direction: column;
+    align-items: left;
+  }
+
+  .item2 {
+    padding-top: 30vh;
+  }
+
+  .item3 {
+    min-height: 30vh;
+    border-bottom-left-radius: 20%;
+  }
+
+  .presentation {
+    font-family: Poppins, sans-serif;
+    font-weight: 400;
+    font-size: 20px;
+    padding-left: 10vw;
+    padding-right: 22vw;
+  }
+
+  .connection {
+    font-family: Poppins, sans-serif;
+    padding-left: 10%;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+    text-decoration: none;
+  }
+
+  .connection2 {
+    font-family: Poppins, sans-serif;
+    padding-left: 20%;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+    text-decoration: none;
+  }
+
+  .color {
+    color: #000 !important;
+  }
+
+  .dot {
+    padding-left: 15px;
+    width: 10px;
+    height: 10px;
+  }
+
+  .span2 {
+    display: flex;
+    justify-content: left;
+    list-style: none;
+    padding-left: 8.5%;
+    margin-bottom: 0;
+  }
+
+  .social-media {
+    margin-left: 26px;
+  }
+
+  .work-section {
+    margin-bottom: 10%;
+  }
+
+  .grid-container {
+    display: grid;
+    gap: 50px;
+    margin-left: 124px;
+    margin-right: 124px;
+    grid-template-columns: repeat(auto-fill, minmax(50vw));
+  }
+
+  .grid-card {
+    padding-bottom: 20px;
+    background-color: #fff;
+    margin-left: 50px;
+    margin-right: 50px;
+    margin-top: 40px;
+    border-radius: 15px;
+    display: flex;
+    flex-direction: row-reverse;
+    border: 3px solid #dfe1e6;
+  }
+
+  .Desktop-version {
```

Comment:
- [ ] pleasei would suggest you to stick to the figma design as much as possible according to The project requirement say  social media icon links are placed in a column in the middle of the page aligned to the right before the text.kindly fix this by align the text and add more padding to make it suit exact like the figma template thank you.

- [ ] your design
![Screenshot_20230506-122914](https://user-images.githubusercontent.com/92458236/236621542-7e20f7fc-6e16-4022-83dc-203a71a88825.jpg)

- [ ] figma design
<img width="181" alt="desktop" src="https://user-images.githubusercontent.com/92458236/236621579-d3b87fb9-c708-4dcb-982f-e2b0f686eefd.png">